udf/mrn_udf_command.cpp

```diff
@@ -53,7 +53,7 @@ struct CommandInfo
 MRN_API mrn_bool mroonga_command_init(UDF_INIT *init, UDF_ARGS *args,
                                       char *message)
 {
-  CommandInfo *info = nullptr;
+  CommandInfo *info = NULL;
```

Comment:
Could you split this from this pull request?
You can push it to the master directly.

Reply:
I see.
I'll push this modification to the master directly and remove it from this PR.